designs/virtualization/4.x/vnc-in-new-window/vnc-in-new-window.md

```diff
@@ -0,0 +1,32 @@
+---
+parent: Virtualization
+version: 4.x
+---
+# VNC in new window
+
+## In the Administrator console
+
+In order to add an option to open the VNC console in a new window, we’ll add a new button ‘Open Console in new window’ to the existing ‘VNC console’ dropdown with the ‘Disconnect before switching’ checkbox option and ‘Send key’ button.
+
+![oprn from the new button](img/Open-console-1c.png)
+
+Another way to open the VNC Console in a new window can be done via the kebab dropdown menu.
+
+![open from the kebab menu](img/Open-console2.png)
```

Comment:
Should it just be "Open Console" and the default behavior is to open in a new window?

Reply:
I guess if we made a change like this it would apply everywhere this option exists...

Reply:
thanks @lizsurette. Shortening the text will look much better.

Reply:
An external link icon might be good to include here as well if this action opens a new window.

Also, reading through the [Actions menu convention](http://openshift.github.io/openshift-origin-design/conventions/documentation/actionmenus.html), maybe `Open Console` should be above `Edit Labels` or even higher? WDYT?

Reply:
Thanks @andybraren +1